rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -200,6 +201,87 @@ lv2_fs_mount_point* lv2_fs_object::get_mp(std::string_view filename)
 	return &g_mp_sys_dev_root;
 }
 
```

Comment:
isn't this what get_mp does? you can simply put their mount point names inside lv2_fs_mount_point instances and use it.

Reply:
Do you mean I should make a change to the `lv2_fs_mount_point` struct?  Because I didn't see it refer to the corresponding VFS entries.

Reply:
Yes, so you would obtain mount point name by reading a member inside it. And this syscall probably doesn't allow paths not containing a single element which is the actual mount point.

Reply:
Actually sorry nvm, I misunderstiod, this function is fine.

Reply:
Although you can call get_mp inside and check which device it is to make this function shorter.

Reply:
> Although you can call get_mp inside and check which device it is to make this function shorter.

I just copied get_mp() and modified the returned values to make get_vfs() lol.

Reply:
> Although you can call get_mp inside it and check which device it is to make this function shorter.

I did it in another way : )